mrmustard/lab_dev/transformations/fockdamping.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The class representing a rotation gate.
+"""
+
+from __future__ import annotations
+
+from typing import Optional, Sequence, Tuple, Union
+
+from .base import Operation
+from ...physics.representations import Bargmann
+from ...physics import triples
+from ..utils import make_parameter, reshape_params
+
+__all__ = ["FockDamping"]
+
+
+class FockDamping(Operation):
+    r"""The Fock damping operator.
+
+    If ``damping`` is an iterable, its length must be equal to `1` or `N`. If it length is equal to `1`,
+    all the modes share the same damping.
+
+    .. code-block ::
+
+        >>> import numpy as np
+        >>> from mrmustard.lab_dev import FockDamping, Coherent
+
+        >>> operator = FockDamping(modes=[0], damping=0.1)
+        >>> input_state = Coherent(modes=[0], x=1, y=0.5)
+        >>> output_state = input_state >> operator
+        >>> assert operator.modes == [0]
+        >>> assert np.allclose(operator.damping.value, [0.1, 0.1])
+        >>> assert output_state.L2_norm < 1
+        
+    Args:
+        modes: The modes this gate is applied to.
+        damping: The damping.
```

Comment:
```suggestion
        damping: The damping factor.
```
If that makes sense?

Reply:
Or parameter even